plugins/Feedback/API.php

```diff
@@ -45,6 +51,10 @@ public function sendFeedbackForFeature($featureName, $like, $message = false)
 
         $body = sprintf("Feature: %s\nLike: %s\n", $featureName, $likeText);
 
+        if (!empty($choice) && $choice != 'undefined') {
```

Comment:
We maybe should try to prefer using `===`/`!==` where easily possible. At least in new code.
```suggestion
        if (!empty($choice) && $choice !== 'undefined') {
```